### ISSUE TITLE
vue Shiny input helper / template

### ISSUE THREAD

**Issue by timelyportfolio** [timelyportfolio/vueR#8]
In `reactR` we have a [helper](https://github.com/react-R/reactR/blob/master/srcjs/input.js) / [template](https://github.com/react-R/reactR/blob/master/R/scaffold_input.R) for creating Shiny-based `react` inputs.  This [article](https://react-r.github.io/reactR/articles/intro_inputs.html) explains how to use the tools.  I think providing similar tools in `vueR` would be helpful for R developers to allow easy integration of `vue` and abstract away some of the intimidating challenges of custom Shiny inputs.

**Comment by JohnCoene** [comment]
Yes exactly, I also have an example of [shiny + vue + vue-bootstrap](https://packer.john-coene.com/#/examples/shiny-bs4) that features an input~ish (`Shiny.setInputValue`). Vue can definitely make those pesky inputs much easier to work with but I also think it makes [custom shiny outputs](https://shiny.rstudio.com/articles/building-outputs.html) more relevant.
